piilo/engines/analyzer.py

```diff
@@ -132,6 +135,15 @@
 #             [entity in egrp and label in lgrp for egrp, lgrp in check_label_groups]
 #         )
 
+# Resolves: 
+# AttributeError: Can't get attribute 'identity' on <module '__main__' >
+class CustomUnpickler(pickle.Unpickler):
+    def find_class(self, module, name):
+        if name == 'identity':
```

Comment:
Does this obviate the need to create a globally scoped identity function? If so, could we just have the following?
```
if name == 'identity':
  return lambda x: x
```